README.md

```diff
@@ -1,17 +1,106 @@
-# Griptape Tool Template
+# Griptape Plugins Template
 
-A Github template repository for creating Griptape tools.
+A Github template repository for creating Griptape plugins.
 
-## Usage
+## Getting Started
 
-This template repository defines the typical structure for a Griptape tool and is meant to be used as a starting point when developing new tools. To create a tool using this template, click the **"Use this template"** button at the top of the repository.
+Via `gh`:
 
-### Steps for creating a new Griptape tool:
+```
+$ gh repo create griptape-plugin-name -p griptape/griptape-plugin-template
+```
+
+Via github web page:
+
+Click on `Use this template`
+
+![](https://docs.github.com/assets/cb-36544/images/help/repository/use-this-template-button.png)
+
+## What is a Griptape Plugin?
+
+A Griptape Plugin is nothing more than a python class that implements one of Griptape's interfaces.
```

Comment:
Start with what it affords, e.g., "A Griptape [extension/plugin] allows you to make your own functionality for your Griptape agents..." Since you are already using the 2nd person elsewhere in this document, the second sentence can be "You create one as a Python class that implements a Griptape interface."

You've told me what one is but now why I would want to use one. Give some concrete examples.

Reply:
I understand the reason behind saying "nothing more than" because it makes it sound simple and easy.. but it ends up sounding not very impressive.  Maybe just simplify?

also, not sure what "implements one of Griptape's interfaces" means. When I hear "interfaces" I think "UI".